packages/suite-native-atoms/src/Switch.tsx

```diff
@@ -1,55 +1,90 @@
-import React from 'react';
-import { TouchableOpacity } from 'react-native';
+import React, { useEffect } from 'react';
+import Animated, {
+    Easing,
+    useAnimatedStyle,
+    useSharedValue,
+    interpolateColor,
+    withTiming,
+} from 'react-native-reanimated';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box } from './Box';
+import { Pressable } from 'react-native';
```

Comment:
nit: I myself try to create some kind of import order structure where
1. react + react native imports
2. other libraries
3. project utils
4. project components

But I don't see it as too important, just a nit

Reply:
I have draft PR where eslint automatically sort imports, but didn't had time to finish it